src/bundle/Resources/public/js/CKEditor/common/input-number/input-number-view.js

```diff
@@ -0,0 +1,33 @@
+import InputTextView from '@ckeditor/ckeditor5-ui/src/inputtext/inputtextview';
+
+export default class InputNumberView extends InputTextView {
+    constructor(locale) {
+        super(locale);
+
+        const bind = this.bindTemplate;
```

Comment:
maybe variable should be named `bindTemplate` after all? `bind` is quite vast term